instrumentation/active_model_serializers/example/standalone.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'bundler/inline'
+
+gemfile(true) do
+  source 'https://rubygems.org'
+
+  gem 'active_model_serializers'
+  gem 'opentelemetry-api'
+  gem 'opentelemetry-common'
+  gem 'opentelemetry-instrumentation-active_model_serializers', path: '../'
+  gem 'opentelemetry-sdk'
+  gem 'opentelemetry-exporter-otlp'
+end
+
+OpenTelemetry::SDK.configure do |c|
+  c.service_name = 'active_model_serializers_example'
+  c.use_all
+end
+
+# without Rails and the Railtie automation, must manually trigger
+# instrumentation subscription after SDK is configured
+OpenTelemetry::Instrumentation::ActiveModelSerializers.subscribe
```

Comment:
I don't love this extra needed step outside the configure block to get the instrumentation to Actually Instrument when Rails wasn't around to detect and call Railtie hooks. I couldn't get the SpanScriber to have a real, configured Tracer when instantiating the subscriber within the instrumentation install step. I see that ActionMailer and ActionView are relying entirely on a Railtie to subscribe at the end of Rails initialization. I figured I would add this interface for folks who may be using ActiveModelSerializers outside of a full Rails app.

Reply:
I understand what you mean. 

Want to do some sort of hacky thing that checks if `Rails` is defined and use the Railtie, otherwise manually install?